ms-patches/off-by-1-AbsPathInImage-fix.yml

```diff
@@ -0,0 +1,8 @@
+title: 'Fix off by 1 error in AbsPathInImage.java Test'

```

Comment:
Q: What is the purpose of including the yml file?

Reply:
This is part of Derek's new official process to include bug fixes in our jdk mirror. This will help with the automation of merging into an official master ms-patches branch